pkg/enemy/types.go

```diff
@@ -0,0 +1,76 @@
+package enemy
+
+import (
+	"fmt"
+
+	"github.com/genshinsim/gcsim/pkg/core/attributes"
+	"github.com/genshinsim/gcsim/pkg/core/curves"
+	"github.com/genshinsim/gcsim/pkg/core/info"
+	"github.com/genshinsim/gcsim/pkg/shortcut"
+)
+
+var abyssHpMultipliers = map[string]float64{
+	"ruinserpent":    2.0,
+	"goldenwolflord": 2.0,
+}
+
+type TargetParams struct {
+	HpMultiplier float64
+	Particles    bool
+}
+
+func ConfigureTarget(profile *info.EnemyProfile, name string, params TargetParams) error {
+	if !(1 <= profile.Level && profile.Level <= 100) {
+		return fmt.Errorf("invalid target level: must be between 1 and 100")
+	}
+	if name == "dummy" {
```

Comment:
This is undocumented, please document it (part where you describe what type does) or remove.